recipes/foamalgo/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "foamalgo" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "135af3991e10d2275f1ad6ff68d1a497e3ad9b7b6cba718d551d1d07cb0e23d7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/zhujun98/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - tbb-devel >=2020.1
+    - numpy
```

Comment:
It seems like this depends on pip, and does it link against numpy?

Reply:
Yes, it uses `pybind11` and `xtensor-python`. And `numpy` is a dependency of `xtensor-python`. And `xtensor-python` is submodule.

Reply:
Please follow the directions here then: https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
Maybe I should add the rest dependencies `xsimd`, `xtensor`,  `xtensor-blas`, ` xtensor-python`, although cmake will download them if they are not found. I will do it tomorrow. Sleepy now zzz...

Reply:
Uggg yeah, you need to do that.

Reply:
This has been fixed! Thank you!